deployments/polygon/usdc/configuration.json

```diff
@@ -24,6 +24,15 @@
     "baseMinForRewards": "1000000e6"
   },
   "assets": {
+    "WMATIC": {
+      "address": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+      "priceFeed": "0xAB594600376Ec9fD91F8e885dADF0CE036862dE0",
```

Comment:
This is the `MATIC / USD` price feed:

https://polygonscan.com/address/0xAB594600376Ec9fD91F8e885dADF0CE036862dE0#readContract